dashboard-api/internal/types/studyconf.go

```diff
@@ -266,6 +270,48 @@ type CreativeConf struct {
 	Tags           []string `json:"tags"`
 }
 
+// AudienceConf is how the audience is configured for the
+// study
+type AudienceConf struct {
+	Name              string             `json:"name"`
+	SubType           string             `json:"subtype"`
+	Lookalike         *Lookalike         `json:"lookalike,omitempty"`
+	QuestionTargeting *QuestionTargeting `json:"question_targeting"`
+	Partioning        *Partioning        `json:"partioning,omitempty"`
+}
+
+type QuestionTargeting struct {
+	OP   string      `json:"op"`
+	Vars []TargetVar `json"vars"`
+}
+
+type TargetVar struct {
+	Type  string `json:"type"`
+	Value string `json:"value"`
```

Comment:
@nandanrao I see in adopt this is:
```python
class TargetVar(BaseModel):
    type: str
    value: Union[str, int, float]
```
However in the database we only seem to have strings for values, is this acceptable or should I just make TargetVars an blank interface?

Reply:
We should leave it as an interface/rawJson because while most times we use a string, there's no reason we couldn't use a number.